saltstack/base/system/files/etc/motd

```diff
@@ -5,5 +5,5 @@ Welcome to
   \____ \|  _ \ / ___) | | | |_/ ) ___ |/ ___)
   _____) ) |_| | |   | |_| |  _ (| ____| |
  (______/|  __/|_|    \__  |_| \_)_____)_|
-         |_|         (____/
+         |_|         (____/ dev
```

Comment:
Why this?

Reply:
to know that you are sshing to dev devvm, do you want me to remove it?

Reply:
Yes, for me it looks wrong. If you want to output some informational data then it should go to the next line and should have a key for it eg. `Environment: development` but I think its not needed at all.

Reply:
ok I will remove it...